### PR TITLE
Replaces the general fabricator in sci with a unique fabricator.

### PR DESCRIPTION
**Pull request by DimWhat**
[qol]
## About the PR 
Replaces the general fab found in almost all science departments with a science fabricator with it's own set of recipes that should be a lot more useful for doing science experiments with. This should make things like mechanical droppers, biosuits, and other science gear reproducable in case somebody nabs them all.
image

I'll figure out a way to add this to atlas in either this PR or another PR later since atlas doesn't have a sci genfab
## Why's this needed? 
Currently sci can get the full access to a bunch of stuff they don't really need for any of their jobs, don't really recall the amount of times I've needed to get my hands on 10 different light bulbs. This will clear up the list of items sci can get while also letting them do more SCIENCE.

## Changelog

```changelog
(u)DimWhat
(+)Replaced the fabricator in science with a unique variant.
```

### CODE REVIEW

**Comment by codeJester27** [comment]
Can you add cable coil and the player vending module?

File: code/datums/manufacturing.dm [hunk 1]
```diff
@@ -1618,9 +1618,43 @@ ABSTRACT_TYPE(/datum/manufacture)
...
+	category = "Clothing"
+
+/datum/manufacture/mechdropper
+	name ="Mechanical Dropper"
+	item_paths = list("MET-1", "CON-1")
+	item_amounts = list(3, 3)
+	item_outputs = list(/obj/item/reagent_containers/dropper/mechanical)
+	time = 3 SECONDS
+	create = 1
+	category = "tool"
```
Comment (Tarmunora) [inline code/datums/manufacturing.dm:1657]:
```suggestion
	category = "Tool"
```

**Review by Tarmunora — APPROVED** [review]

**Comment by ThatFiveGuys** [comment]
Add it to the cargo crate please

File: code/obj/machinery/manufacturer.dm [hunk 2]
```diff
@@ -2175,6 +2175,39 @@
...
+	free_resources = list(/obj/item/material_piece/steel,
+		/obj/item/material_piece/copper,
+		/obj/item/material_piece/glass,
+		/obj/item/material_piece/cloth/cottonfabric)
+	available = list(
+		/datum/manufacture/screwdriver,
+		/datum/manufacture/wirecutters,
+		/datum/manufacture/wrench,
+		/datum/manufacture/crowbar,
+		/datum/manufacture/extinguisher,
```
Comment (ZeWaka) [inline code/obj/machinery/manufacturer.dm:2193]: don't think these are really necessary
really trivializes general manufacturers

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**